code/modules/clothing/clothing.dm

```diff
@@ -359,6 +360,7 @@ BLIND     // can't see anything
 	slowdown = SHOES_SLOWDOWN
 	var/blood_state = BLOOD_STATE_NOT_BLOODY
 	var/list/bloody_shoes = list(BLOOD_STATE_HUMAN = 0,BLOOD_STATE_XENO = 0, BLOOD_STATE_OIL = 0, BLOOD_STATE_NOT_BLOODY = 0)
+	var/dropped_twice = FALSE

```

Comment:
@optimumtact it's to avoid dropping the shoe from your hand and it still shifts ur pixel down